client/src/modules/notification/components/Notification.jsx

```diff
@@ -4,9 +4,13 @@ import { FormattedMessage } from 'react-intl';
 
 import MessageBox from 'common/components/MessageBox';
 import { MessageType as NotificationType } from 'common/constants/enums';
+import { mapObject } from 'common/utils/helpers';
 
 const Notification = ({ data, id, redirect, type }) => {
-  const values = typeof data === 'string' ? { data } : { ...data };
+  const values =
+    typeof data === 'string'
+      ? { data: <em>{data}</em> }
+      : { ...mapObject(value => <em>{value}</em>)(data) };
```

Comment:
This can be simplified to:
```javascript
const wrap = value => <em>{value}</em>;
const dataSource = typeof data === 'string' ? { data } : data;
const values = mapObject(wrap)(dataSource);
```